neurons/validator/app.py

```diff
@@ -11,8 +11,10 @@
 
 credentials = open("streamlit_credentials.txt", "r").read()
 credentials_split = credentials.split("\n")
-username = credentials_split[0].split("username=")[1]
-password = credentials_split[1].split("password=")[1]
+
+hashkey = credentials_split[0].split("hashkey=")[1]
```

Comment:
If we're going to add this do we not need to write a hash key to the stremlit_credentials.txt file here: https://github.com/Supreme-Emperor-Wang/ImageAlchemy/blob/b9cbfa71c42ba4a2ed81c46cd3d77093380cb4e2/neurons/validator/validator.py#L193